src/verilator/VerilatorObject.py

```diff
@@ -0,0 +1,14 @@
+from m5.params import *
+from m5.SimObject import SimObject
+
+class VerilatorObject(SimObject):
+    type = 'VerilatorObject'
+    cxx_header = "verilator/verilator_object.hh"
+
+    cycles = Param.Int(0, "Maximum number of cycles to simulate,
```

Comment:
I don't think you need this option here. Instead, this is a parameter to m5.simulate() in the runscript.